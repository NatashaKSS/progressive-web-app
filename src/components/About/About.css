.title {
  font-size: 1.8rem;
}

.container {
  padding: 1rem 2rem;
}

.content {
  font-size: 1.5rem;
  line-height: 1.5;
}

.link {
  color: #2331de;
}

.link:focus,
.link:hover {
  color: #7e88fc;
  cursor: pointer;
}
