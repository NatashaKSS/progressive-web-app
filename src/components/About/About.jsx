import React from 'react';

import styles from './About.css';

const About = () => {
  return (
    <div className={styles.container}>
      <h1 className={styles.title}>About</h1>
      <p className={styles.content}>Blah blah blah</p>
    </div>
  );
}

export default About;
