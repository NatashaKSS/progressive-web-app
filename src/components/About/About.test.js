import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from './About';

test('<About /> should render as expected', () => {
  const component = shallow(<About />);
  const tree = toJson(component);
  // expect(component.contains("About")).toBe(true);
  expect(tree).toMatchSnapshot();
});
