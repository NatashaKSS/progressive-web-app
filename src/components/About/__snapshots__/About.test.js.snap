// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render as expected 1`] = `
<div
  className="container"
>
  <h1
    className="title"
  >
    About
  </h1>
  <p
    className="content"
  >
    Blah blah blah
  </p>
</div>
`;
